### PR TITLE
Make /u unbroadcastable

### PR DESCRIPTION
**Pull request by ghost**
Makes it so nsfw stuff doesnt plague the chat.

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
Eh as much as that is probably a good idea.... I would prefer disabling broadcasting in Lobby perhaps

**Comment by DeathlyPlays** [comment]
Oooooo we could have it set as a room option whether they wish to disable /u access from being broadcasted etc if you wanna get really complicated

**Comment by AlphaWind** [comment]
@DeathlyPlays would this make more sense? `if (room.id === 'lobby') return this.errorReply("This command cannot be used in the Lobby.");`

**Comment by DeathlyPlays** [comment]
Actually no @AlphaWind usage of the command is fine in Lobby as long as it doesn't pollute chat in general (via broadcasting imo)

File: server-plugins/server.js [hunk 1]
```diff
@@ -319,6 +319,7 @@ exports.commands = {
 	u: 'urbandefine',
 	ud: 'urbandefine',
 	urbandefine: function (target, room, user) {
+		if (this.runBroadcast() && room.id === 'lobby') return this.sendReply("Minors use Pokemon Showdown so it is probably a bad idea to broadcast this command.");
```
Comment (DeathlyPlays) [inline server-plugins/server.js:322]: Thank for doing this later expansion will be a disable/enable for a room

**Pull request merged**